index.html

```diff
@@ -325,6 +329,8 @@ <h2>Glossary</h2>
 	
 <p><dfn data-lt="producer|producers">Producer</dfn>. When talking about strings on the Web, the W3C Internationalization group refers to a producer as any process where natural language string data is created for later storage, processing, or interchange.</p>
 
+<p><dfn data-lt="paragraph direction">Paragraph direction</dfn>. The initial <a>base direction</a>  of  a paragraph or string, which resolves to either <em>left-to-right</em> or <em>right-to-left</em>. The paragraph direction may be overridden by nested, embedding controls that change the  direction for an inline range of text, but the paragraph direction sets the starting point which the <a>Unicode Bidirectional Algorithm</a> uses  to calculate the direction of the embedded levels. For more details, see <a href="http://www.unicode.org/reports/tr9/">Unicode Standard Annex #9, “Unicode Bidirectional Algorithm,</a>” especially definitions BD2–BD5.<br><span class="see-also">See also the definition in the <a href="https://www.unicode.org/glossary/#paragraph_direction">Unicode Standard</a>.</span></p>
```

Comment:
Note: you only need `data-lt=` for when term is different. Also, ReSpec handles plurals for you, so things like "producer|producers" is not needed.

Reply:
thanks Marcos. So no need to worry about capital letters?

Reply:
Nope. Also handled :)

Reply:
Nits: fix double spaces. Add [UTR9] ref after the linked text. I would probably tag the definitions with <code> or one of our similar styles to set the definitions out.

Substantive comments:

I would probably clarify the nested stuff slightly like so:

> Nested embedding controls can be used to change the direction for an inline range of text, but the paragraph direction sets the starting point which the <a>Unicode Bidirectional Algorithm</a> uses to calculate the direction of the embedded levels.

Reply:
Done.